tests/Jenkins/utilities.sh

```diff
@@ -333,16 +324,15 @@ installDIRAC() {
     rm "installer.sh"
     # TODO: Remove
     echo "source \"$PWD/diracos/diracosrc\"" > "$PWD/bashrc"
+    echo "export X509_CERT_DIR=\"$PWD/diracos/etc/grid-security/certificates\"" >> "$PWD/bashrc"
     source diracos/diracosrc
     if [[ -n "${DIRAC_RELEASE+x}" ]]; then
       if [[ -z "${ALTERNATIVE_MODULES}" ]]; then
         pip install "${DIRAC_RELEASE}"
       fi
     fi
     for module_path in "${ALTERNATIVE_MODULES[@]}"; do
-      # ALTERNATIVE_MODULES can be a list of URLs to pip-installable modules
-      # or something like git+https://github.com/fstagni/DIRAC.git@v7r2-fixes33#egg=DIRAC[pilot]
-      pip install "${module_path}"
+      pip install ${PIP_INSTALL_EXTRA_ARGS:-} "${module_path}"
```

Comment:
Sorry, can you explain what is `PIP_INSTALL_EXTRA_ARGS` ?

Reply:
At the moment it's only used to pass `-e` if there is a tty attached (i.e. not in CI) so you can edit the code in the services without re-installing or manually copying back and forth with the container. The default can be overridden by passing `--[no-]editable` and it was very convenient when fixing the Python 3 server installation.